python/pyspark/sql/tests/test_functions.py

```diff
@@ -67,7 +67,7 @@ def test_function_parity(self):
 
         # For functions that are named differently in pyspark this is the mapping of their
         # python name to the JVM equivalent
-        py_equiv_jvm = {"create_map": "map"}
+        py_equiv_jvm = {"create_map": "map", "py_len": "len"}
```

Comment:
I think we don't need this change any more

Reply:
+1